api/src/pcapi/routes/backoffice/external/zendesk_sell/blueprint.py

```diff
@@ -0,0 +1,128 @@
+                f'offerer_id: {item["custom_fields"][zendesk_sell.ZendeskCustomFieldsShort.PRODUCT_OFFERER_ID.value]}, '
+                f'siren: {item["custom_fields"][zendesk_sell.ZendeskCustomFieldsShort.SIREN.value]}',
+                "warning",
+            )
+        return redirect(url, code=303)
+    except zendesk_sell.ContactNotFoundError:
+        flash("La structure n'a pas été trouvée dans zendesk sell", "warning")
+        return redirect(url, code=303)
+    except requests.exceptions.HTTPError as http_error:
+        flash(f"Une erreur {http_error.response} s'est produite : {http_error}", "warning")
+        return redirect(url, code=303)
+
+    try:
+        offerer_zendesk_id = zendesk_offerer_data["id"]
+        zendesk_sell_backend.update_offerer(offerer_zendesk_id, offerer)
+    except requests.exceptions.HTTPError as http_error:
+        flash(f"Une erreur {http_error.response} s'est produite : {http_error}", "warning")
+        return redirect(url, code=303)
+
+    flash("La mise a jour a été effectuée avec succès", "success")
+    return redirect(url, code=303)
+
+
+@zendesk_sell_blueprint.route("/venue/<int:venue_id>/update", methods=["POST"])
+def update_venue(venue_id: int) -> utils.BackofficeResponse:
+    venue = offerers_models.Venue.query.get_or_404(venue_id)
+    url = url_for("backoffice_web.venue.get", venue_id=venue_id)
+
+    if venue.isVirtual or not venue.isPermanent:
+        flash("Ce lieu est virtuel ou non permanent", "warning")
+        return redirect(url, code=303)
+
+    try:
+        zendesk_venue_data = zendesk_sell_backend.get_venue_by_id(venue)
+    except zendesk_sell.ContactFoundMoreThanOneError as e:
+        flash("Il y a plusieurs lieux avec cet id dans Zendesk Sell", "warning")
+        for item in e.items:
+            flash(
+                f'    ==> sell id: {item["id"]}, '
+                f'venue_id: {item["custom_fields"][zendesk_sell.ZendeskCustomFieldsShort.PRODUCT_VENUE_ID.value]}, '
+                f'siret: {item["custom_fields"][zendesk_sell.ZendeskCustomFieldsShort.SIRET.value]}',
+                "warning",
+            )
+        return redirect(url, code=303)
+    except zendesk_sell.ContactNotFoundError:
+        flash("Le lieu n'a pas été trouvé", "warning")
+        return redirect(url, code=303)
+    except requests.exceptions.HTTPError as http_error:
+        flash("Une erreur s'est produite " + str(http_error) + ", " + str(http_error.response), "warning")
+        return redirect(url, code=303)
+
+    try:
+        zendesk_venue_id = zendesk_venue_data["id"]
+        parent_organization_id = _get_parent_organization_id(venue)
+        if parent_organization_id == zendesk_sell.SEARCH_PARENT:
+            raise zendesk_sell.ContactNotFoundError
+        zendesk_sell_backend.update_venue(zendesk_venue_id, venue, parent_organization_id)
+    except zendesk_sell.ContactNotFoundError:
+        flash("La structure parente n'a pas été trouvée dans zendesk sell", "warning")
```

Comment:
Anecdotique mais plus propre : les majuscules : 
```suggestion
        flash("La structure parente n'a pas été trouvée dans Zendesk Sell", "warning")
```